dapp/src/utils/contracts.js

```diff
@@ -128,9 +129,32 @@ export async function setupContracts(account, library, chainId) {
     }
   }
 
+  const fetchCreditsPerToken = async () => {
+    const response = await fetch(process.env.CREDITS_ANALYTICS_ENDPOINT)
+    if (response.ok) {
+      const json = await response.json()
+      YieldStore.update((s) => {
+        s.currentCreditsPerToken = parseFloat(json.current_credits_per_token)
+        s.nextCreditsPerToken = parseFloat(json.next_credits_per_token)
+      })
+    }
+  }
+
+  const fetchCreditsBalance = async () => {
+    if (!walletConnected) {
+      return
+    }
+    const credits = await ousd.creditsBalanceOf(account)
+    AccountStore.update((s) => {
+      s.creditsBalanceOf = ethers.utils.formatUnits(credits[0], 18)
+    })
+  }
+
   const callWithDelay = (fetchAPR = false) => {
     setTimeout(async () => {
       fetchExchangeRates()
+      fetchCreditsPerToken()
+      fetchCreditsBalance()
       if (fetchAPR) {
```

Comment:
I think we should drop this condition and always fetch the APR so that it updates along with the pending rebase. Any reason not to @sparrowDom @DanielVF? If it results in too many hits on the analytics server, we could just consolidate to one endpoint and put everything in the payload.

Reply:
Analytics server caches things like this - you can beat on it with a million people and still only works a couple times a minute.

Reply:
Anything I have to do regarding this?

Reply:
This condition was in place in order to save on Alchemy calls. Now that we query the backend analytics this can be removed.

Reply:
Resolved by 340352e ✔️